google/cloud/storage/blob.py

```diff
@@ -804,7 +804,7 @@ def _extract_headers_from_download(self, response):
 
         digests = {}
         for encoded_digest in x_goog_hash.split(","):
-            match = re.match(r"(crc32c|md5)=([\w\d]+)==", encoded_digest)
+            match = re.match(r"(crc32c|md5)=([\w\d/]+==)", encoded_digest)
```

Comment:
Rather than hard-wire the two character padding, we can have it work for any length (`base64` adds up to three padding bytes to make the encoded string length a multiple of 4):
```suggestion
            match = re.match(r"(crc32c|md5)=([\w\d]+={0,3})", encoded_digest)
```